### PR TITLE
add acceptance tests with real vpn client/server setup

### PR DESCRIPTION
**Pull request by Dan33l**
#### Pull Request (PR) description

Creates a client / server setup with SUT. This permit to make more tests.

#### This Pull Request (PR) fixes the following issues

    test issues like #225

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,14 @@ Puppet module to manage OpenVPN servers and clients.
...
-* Amazon
 
 ## Dependencies
   - [puppetlabs-concat 3.0.0+](https://github.com/puppetlabs/puppetlabs-concat)
   - [puppetlabs-stdlib 4.25.0+](https://github.com/puppetlabs/puppetlabs-stdlib)
 
 ## Puppet
 
-* Version >= 4.7.1
+* Version >= 4.25.0
```
Comment (alexjfisher) [inline README.md]: ??

Comment (Dan33l) [reply README.md]: I updated the README and saw this is not uptodate with metadata.json

File: README.md [hunk 2]
```diff
@@ -180,18 +177,16 @@ OpenVPN was setting its default value to `BF-CBC`. In newer versions of OpenVPN
...
 
 ##### Why
 
-Details of these ciphers and their uses can be found in the documentation links
-above.
-
+Details of these ciphers and their uses can be found in the documentation links above.
 
+Note : TLS ciphers suites shipped with OSes ubuntu14.04 and debian8 are too old compared to our default values.
+If these OSes hosts the server with clients more moderns, you will probably have to use custom value for option `tls_cipher`.
```
Comment (alexjfisher) [inline README.md]: This sentence doesn’t read too well.

File: spec/acceptance/openvpn_spec.rb [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  crt_path = '/etc/openvpn/test_openvpn_server/easy-rsa/keys/issued'
+  index_path = '/etc/openvpn/test_openvpn_server/easy-rsa/keys'
+when 'Debian'
+  server_crt = '/etc/openvpn/test_openvpn_server/easy-rsa/keys/server.crt'
+  key_path = '/etc/openvpn/test_openvpn_server/easy-rsa/keys'
+  crt_path = '/etc/openvpn/test_openvpn_server/easy-rsa/keys'
+  index_path = '/etc/openvpn/test_openvpn_server/easy-rsa/keys'
+end
+
+# All-terrain tls ciphers are used to be able to works with all supported OSes.
```
Comment (alexjfisher) [inline spec/acceptance/openvpn_spec.rb]: ‘work’ instead of ‘works’

File: spec/acceptance/openvpn_spec.rb [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+# Default value is with ciphers too recents for old OSes like ubuntu 14.04.
```
Comment (alexjfisher) [inline spec/acceptance/openvpn_spec.rb]: ‘recent’ instead of ‘recents’

**Pull request merged**